dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -2199,13 +2963,37 @@ FolderOptionsFileTypesDlg(
                         SHOpenWithDialog(hwndDlg, &Info);
                     }
                     break;
+
+                case IDC_FILETYPES_ADVANCED:
+                    edittype.hwndLV = GetDlgItem(hwndDlg, IDC_FILETYPES_LISTVIEW);
+                    edittype.pEntry = GetListViewEntry(edittype.hwndLV);
+                    DialogBoxParamW(shell32_hInstance, MAKEINTRESOURCEW(IDD_EDITTYPE),
+                                    hwndDlg, EditTypeDlgProc, (LPARAM)&edittype);
+                    break;
             }
             break;
 
         case WM_NOTIFY:
             lppl = (LPNMLISTVIEW) lParam;
             switch (lppl->hdr.code)
             {
+                case LVN_KEYDOWN:
+                    {
```

Comment:
Broken indentation. Braces should be aligned with 'c' of "case", and then the code block inside uses a 4-space indent.